perception/traffic_light_occlusion_predictor/src/nodelet.cpp

```diff
@@ -72,12 +74,22 @@ TrafficLightOcclusionPredictorNodelet::TrafficLightOcclusionPredictorNodelet(
     config_.elevation_occlusion_resolution_deg);
 
   const std::vector<std::string> topics{
-    "~/input/traffic_signals", "~/input/rois", "~/input/camera_info", "~/input/cloud"};
+    "~/input/traffic_signals_car", "~/input/rois", "~/input/camera_info", "~/input/cloud"};
   const std::vector<rclcpp::QoS> qos(topics.size(), rclcpp::SensorDataQoS());
   synchronizer_ = std::make_shared<SynchronizerType>(
     this, topics, qos,
-    std::bind(&TrafficLightOcclusionPredictorNodelet::syncCallback, this, _1, _2, _3, _4),
+    std::bind(&TrafficLightOcclusionPredictorNodelet::syncCallback, this, _1, _2, _3, _4, 0),
```

Comment:
```suggestion
    std::bind(&TrafficLightOcclusionPredictorNodelet::syncCallback, this, _1, _2, _3, _4, tier4_perception_msgs::msg::TrafficLightRoi::CAR_TL),
```
to avoid the use of magic number

Reply:
fixed in
https://github.com/autowarefoundation/autoware.universe/commit/2f237e8a644b8b7ce191b74196f7a900e027bb80